cypress/fixtures/suite/yves/core/static-locale-switching.json

```diff
@@ -0,0 +1,6 @@
+{
+  "availableLocales": ["de", "en"],
+  "localeDE": "de_DE",
+  "localeEN": "en_US",
```

Comment:
what about static fixtures for other repositories (b2b, b2c ...)?

for productName - left comment above -> use dynamic instead